docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
 
   # main clowder application
   clowder:
-    image: clowder/clowder:${CLOWDER_VERSION:-latest}
+    image: clowder:bugfix
```

Comment:
Again this looks like a temp commit, should it revert back to the original image?